lib/components/molecules/Accordion/Accordion.js

```diff
@@ -0,0 +1,51 @@
+// @flow
+/**
+ *
+ * Accordion
+ *
+ */
+import React from 'react';
+import type { Node } from 'react';
+import styled from 'styled-components';
+import styles from './Accordion.style';
+import AccordionHeader from './AccordionHeader';
+import AccordionPanel from './AccordionPanel';
+
+type Props = {
+  className?: string,
+  accordionContent?: [],
+  isOpen?: boolean,
+};
+
+// eslint-disable-next-line no-unused-vars
+const Accordion = ({ className, accordionContent, isOpen }: Props): Node => (
+  <div className={className}>
+    {accordionContent &&
+      accordionContent.map((item, index) => (
+        <details
+          className="accordion-item"
+          key={`accordion-${index.toString()}`}
+          open={item.isOpen}
+        >
+          <AccordionHeader className={item.headerClassName || ''}>
```

Comment:
AccordionHeader and AccordionPanel should be exposed outside as compound components so that users of accordion can define their bodies in decorative manner.